src/ui/handlers/egg-gacha-ui-handler.ts

```diff
@@ -22,7 +22,7 @@ import i18next from "i18next";
 import { MessageUiHandler } from "./message-ui-handler";
```

Comment:
NIT: relative path (since you are refactoring those here anyway 👍 

```suggestion
import { MessageUiHandler } from "#app/ui/handlers/message-ui-handler";
```